pkg/storegateway/bucket_e2e_test.go

```diff
@@ -889,3 +892,33 @@ func foreachStore(t *testing.T, runTest func(t *testing.T, newSuite suiteFactory
 		runTest(t, factory)
 	})
 }
+
+func toLabels(t *testing.T, labelValuePairs []string) (result labels.Labels) {
+	if len(labelValuePairs)%2 != 0 {
```

Comment:
should these be denoted as `t.Helper()` functions?

Reply:
looking at how the Helper is used I think you should identify each Helper function as one in order to skip it https://github.com/golang/go/blob/069f9fb20548e904cb94f165f4cbf716a0fc108e/src/testing/testing.go#L688-L762

I think this code starts iterating the stack from the closest caller on the stack and then stops when it either finds a non-helper function or the function of the test itself. So I believe that once it encounters a non-helper function it will not skip the frame even though a parent function is a Helper function.

Reply:
Hey,
Thanks for the reference. Also did a bit of reference and I agree, it makes sense to add `t.Helper` in this case.